packages/playwright-core/src/cli/cli.ts

```diff
@@ -682,8 +682,8 @@ function logErrorAndExit(e: Error) {
   process.exit(1);
 }
 
-function language(): string {
-  return process.env.PW_LANG_NAME || 'test';
+function codegenId(): string {
+  return process.env.PW_LANG_NAME || 'javascript-test';
```

Comment:
isn't it by definition more TS? Or did we make "import" work in JS well?

Reply:
Hm, I did not think it about it. import does work in JS, but I wonder if we should differentiate between JS and TS now...

Reply:
tested with js/ts/js-esm all good, lgtm.